packages/connect/src/api/cardano/api/cardanoSignMessage.ts

```diff
@@ -0,0 +1,118 @@
+import { AbstractMethod } from '../../../core/AbstractMethod';
+import { PROTO, CARDANO, ERRORS } from '../../../constants';
+import { getFirmwareRange } from '../../common/paramsValidator';
+import { getMiscNetwork } from '../../../data/coinInfo';
+import { Path } from '../cardanoInputs';
+import { validatePath } from '../../../utils/pathUtils';
+import { hexStringByteLength, sendChunkedHexString } from '../cardanoUtils';
+import {
+    CardanoSignMessage as CardanoSignMessageSchema,
+    CardanoMessageHeaders,
+    CardanoSignedMessage,
+} from '../../../types/api/cardano';
+import { addressParametersToProto } from '../cardanoAddressParameters';
+import { Assert } from '@trezor/schema-utils';
+import { hasHexPrefix, isHexString } from '../../../utils/formatUtils';
+
+export type CardanoSignMessageParams = {
+    signingPath: Path;
```

Comment:
Small nitpick, but everywhere else we just call this param `path`

Reply:
Originally it was named as such because we supported supplying a different path (then called `keyPath`) to be used for the `address` header field instead of address parameters. Then we opted to drop it, but I left the name there in case we ever needed to add the second path back. But I can rename it to `path`, if you think it would be nicer